### PR TITLE
add info on bug bounty invoice requirements

### PR DESCRIPTION
**Pull request by PierreBesson**

### CODE REVIEW

File: pages/bug-bounties.md [hunk 1]
```diff
@@ -45,5 +45,12 @@ In order to claim the money, you must:
 - In order to close the ticket automatically, you must have one commit message with the `Fix` keyword. For example, `Fix #1234` to close ticket `#1234`.
 - That Pull Request must be merged by someone from the core team. If there are several Pull Requests, the core team member either selects the most recent one or the best one - that's up to him to decide what is best for the project.
 - You can then [add a $100 expense on the JHipster OpenCollective](https://opencollective.com/generator-jhipster/expenses/new). You must add a link to your Pull Request in the description. For example: `$100 bug bounty claim for https://github.com/jhipster/generator-jhipster/pull/1234`.
+
+<div class="alert alert-error"><i>Tip: </i>
+
+**Important: In this step you have to provide an Invoice. You will be provided a Google Docs template to help you do it. See the [Open Collective FAQ for expenses](https://opencollective.com/faq/expenses) for more details.**
```
Comment (jdubois) [inline pages/bug-bounties.md]: "Invoice" -> "invoice"
And why do you say that people will be provided a Google Docs template? There's an example in Open Collective, but we don't provide anything

**Comment by PierreBesson** [comment]
Because there is now a link to the invoice template on the opencollective
"submit an expense" page. I just left work. Can you fix the typo for me
please.

**Comment by jdubois** [comment]
@PierreBesson yes I'll have a look

**Comment by jdubois** [comment]
Argh the tip does not render correctly, I'll just add a bullet point instead

**Comment by PierreBesson** [comment]
Sorry

**Comment by jdubois** [comment]
No problem, I know the formatting system sucks

**Pull request merged**